managed/CustomGroup_Currency_Exchange_rates.mgd.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+return [
+  [
+    'name' => 'CustomGroup_Currency_Exchange_rates',
+    'entity' => 'CustomGroup',
+    'cleanup' => 'unused',
+    'update' => 'always',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'Currency_Exchange_rates',
+        'title' => 'Currency Exchange rates',
```

Comment:
you should also add the table_name param so we have more control over the table name that will be created for this custom group.

Reply:
the table_name param seems to be problematic on install/uninstall

```
(
    [entity] => Extension
    [action] => install
    [params] => Array
        (
            [keys] => Array
                (
                    [0] => io.compuco.financeextras
                )

            [debug] => 1
            [version] => 3
        )

    [result] => Array
        (
            [error_code] => unknown error
            [trace] => #0 /Applications/MAMP/htdocs/compu/profiles/compuclient/modules/contrib/civicrm/vendor/pear/pear-core-minimal/src/PEAR.php(922): CRM_Core_Error::exceptionHandler(Object(DB_Error))
#1 /Applications/MAMP/htdocs/compu/profiles/compuclient/modules/contrib/civicrm/vendor/pear/db/DB.php(997): PEAR_Error->__construct('DB Error: unkno...', -1, 16, Array, 'ALTER TABLE fe_...')
```

Removed

Reply:
@olayiwola-compucorp 

Then can I ask you to add this custom group and fields like this way instead: https://github.com/compucorp/io.compuco.multicompanyaccounting/pull/2/commits/fd8edb98a7e2fa7ca176da4bcd8652533f5f1ead#diff-1606911e7602e0fbb9fb4656dcb73aa7cb8782ac0793e69ef425e026d1c6d4df

we might in the future need to use this custom group in SSP or Drupal views or maybe to do some complex queries , so not having predictability in the table name will make such tasks complicated. Also Managed entities are not something I like to use in general, they often resulted in deployment failures in the past and I doubt they are any better these days.